src/utils.js

```diff
@@ -1,3 +1,7 @@
+function isNode() {
+  return typeof window == "undefined";
+}
```

Comment:
Might be worth considering making this a constant, since the `isNode()` call is more noisy and can't ever change during an execution.
```suggestion
const isNode = typeof window == "undefined";
```